build-farm/make-adopt-build-farm.sh

```diff
@@ -28,18 +28,32 @@ JAVA_FEATURE_VERSION=$(echo "${JAVA_TO_BUILD}" | tr -d "[:alpha:]")
 
 if [ -z "${JAVA_FEATURE_VERSION}" ]
 then
-  # Use Adopt API to get the JDK Head number
-  echo "This appears to be JDK Head. Querying the Adopt API to get the JDK HEAD Number (https://api.adoptopenjdk.net/v3/info/available_releases)..."
-  JAVA_FEATURE_VERSION=$(curl -v https://api.adoptopenjdk.net/v3/info/available_releases | awk '/tip_version/{print$2}')
-    
-  # Checks the api request was successfull and the return value is a number
-  if [ -z "${JAVA_FEATURE_VERSION}" ] || ! [[ "${JAVA_FEATURE_VERSION}" -gt 0 ]]
-  then
-    echo "Failed to query or parse the adopt api. Dumping headers via curl -v https://api.adoptopenjdk.net/v3/info/available_releases..."
-    curl -v https://api.adoptopenjdk.net/v3/info/available_releases
-    exit 1
-  fi
-  echo "JAVA_FEATURE_VERSION is ${JAVA_FEATURE_VERSION}"
+    retryCount=1
+    retryMax=5
+    until [ "$retryCount" -ge "$retryMax" ]
```

Comment:
Bear in mind that with `-ge` instead of `-gt` this will attempt the retrieval a maximum of four times as it won't run through when `retryCount=5`, which is OK but inconsistent with the comment in the PR description

Reply:
Will update the comment :)